inst/extdata/basic-curly.md

```diff
@@ -0,0 +1,9 @@
+---
+title: basic curly
+---
+
+# preface {#pre-face .unnumbered}
+
+hello
+
+I like {xml2} but of course {tinkr} is even cooler!
```

Comment:
One of the difficulties with this approach is that it will also capture and ignore image alt text, which _should_ be sent to the API, whereas the other attributes should be left untouched. 

```suggestion
I like {xml2} but of course {tinkr} is even cooler!

Images that use pandoc style will have curlies with content that should be translated and should be protected.

![a pretty kitten](https://placekitten.com/200/300){#kitteh alt='a picture of a kitten'}

```